docs/docs/guides/web3_providers_guide/index.md

```diff
@@ -101,7 +101,7 @@ As stated above, the injected provider should be in compliance with [EIP-1193](h
 The web3.js 4.x Provider specifications are defined in [web3 base provider](https://github.com/ChainSafe/web3.js/blob/4.x/packages/web3-types/src/web3_base_provider.ts) for Injected Providers.
 
 ```html
-<script src="https://cdn.jsdelivr.net/npm/web3@4.0.1-rc.1/dist/web3.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/web3@4.1.2/dist/web3.min.js"></script>
```

Comment:
@Pumpkir Thanks, 
I think also add in documentation that replace this URL with latest release link listed in: https://www.jsdelivr.com/package/npm/web3

Reply:
> @Pumpkir Thanks, I think also add in documentation that replace this URL with latest release link listed in: https://www.jsdelivr.com/package/npm/web3

Good idea @jdevcs ,
I also found that you can get the last major version like this:
https://cdn.jsdelivr.net/npm/web3/dist/web3.min.js
And the last minor like this:
https://cdn.jsdelivr.net/npm/web3@4/dist/web3.min.js

So we can use one of the above and do not add in the documentation a note asking the reader to replace the Url. However, which one, of the above Url, do you recommend using?

Reply:
https://cdn.jsdelivr.net/npm/web3/dist/web3.min.js is more good

Reply:
Of course, I found two and have completed the replacement